kano/utils/hardware.py

```diff
@@ -112,3 +138,26 @@ def get_mac_address():
     mac_addr_str = mac_addr.upper()
     if len(mac_addr_str) == 17:
         return mac_addr_str
+
+
+def has_min_performance(score):
+    """
+    Check if the hardware we're running on has a minimum given performance.
+
+    Args:
+        score (int) - A performance score just like the ones in PERFORMANCE_SCORES
+
+    Returns:
+        rv (bool) - True if the hardware has a higher score than the given or if the
+                    hardware could not be detected; and False otherwise
+    """
+    rv = True
+    model = get_rpi_model()
+
+    if model in PERFORMANCE_SCORES:
+        model_score = PERFORMANCE_SCORES[model]
+
+        if model_score < score:
+            rv = False
+
+    return rv
```

Comment:
Could this just be

```python
model = get_rpi_model()
model_score = PERFORMANCE_SCORES.get(model, score)

return model_score >= score
```

or, if you don't like `.get` with default values and to make the behaviour when the board is not detected, return early

```python
model = get_rpi_model()

if model not in PERFORMANCE_SCORES:
    return True

model_score = PERFORMANCE_SCORES[model]

return model_score >= score
```